runtime/macros/maze/mazeclean.c

```diff
@@ -9,6 +9,7 @@
 char *M, A, Z, E = 40, line[80], T[3];
 int
 main (C)
+int C;
```

Comment:
Err, I think we should use the normal ANSI C function definition here:
```suggestion
main (int C)
```

Reply:
I was keeping with the "work with all compilers" comment, but yeah I think in 2024 we're safe with using 1989 C :D